furms-web-ui/src/main/java/io/imunity/furms/ui/config/BaseRedirectController.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020 Bixbit s.c. All rights reserved.
+ * See LICENSE file for licensing information.
+ */
+
+package io.imunity.furms.ui.config;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import static io.imunity.furms.ui.constant.LoginFlowConst.LOGIN_URL;
+
+@Controller
+public class BaseRedirectController {
+	@RequestMapping("/")
```

Comment:
@bihu1 question: what will happen if I enter https://localhost:3443/?showSignInOptions.
Will the showSignInOptions parameter be honored?

Reply:
This controller is not available if you are not log in. Spring security has priority.